### PR TITLE
forward falco alerts

### PR DESCRIPTION
**Pull request by Issif**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind user-interface

/kind content

> /kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area videos

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by LucaGuerra** [comment]
I like this! Since it talks mainly about Falcosidekick, perhaps we could change the title to "Forwarding Alerts with Falcosidekick"? cc @leogr @aijamalnk  . Then (after this PR) we could tie it all together from the main Alerts page so that it will serve as a clear path for adopter to pick their preferred alerts integration

File: data/en/features.yaml [hunk 1]
```diff
@@ -12,6 +12,12 @@
   - Runtime detection is a fundamental layer of defense against security blind spots and zero-day bugs in your software supply chain.
   - Streaming approach enables real-time response while minimizing storage costs and complexity.
   - Ready out-of-the-box with rules, which you can customize for your environment.
+- title: Integration with 50+ Systems
+  description: Forward Falco alerts to any off-host SIEM and data lake system for analysis, storage or reaction.
+  icon: /icons/arrow-right.svg
+  features: 
+  - Falco alerts can easily be forwarded to more than 50+ third parties
+  - The JSON format for alerts allows to easily store, analyze or trigger reactions
```
Comment (leogr) [inline data/en/features.yaml:20]:
```suggestion
  - Falco alerts can easily be forwarded to more than 50+ third parties
  - The JSON format for alerts allows for storing, analysis, or triggering reactions easily
```

File: data/en/features.yaml [hunk 2]
```diff
@@ -12,6 +12,12 @@
   - Runtime detection is a fundamental layer of defense against security blind spots and zero-day bugs in your software supply chain.
   - Streaming approach enables real-time response while minimizing storage costs and complexity.
   - Ready out-of-the-box with rules, which you can customize for your environment.
+- title: Integration with 50+ Systems
+  description: Forward Falco alerts to any off-host SIEM and data lake system for analysis, storage or reaction.
```
Comment (leogr) [inline data/en/features.yaml:16]:
```suggestion
  description: Forward Falco alerts to any off-host SIEM and data lake system for analysis, storage, or reaction.
```

File: content/en/docs/alerts/_index.md [hunk 3]
```diff
@@ -14,6 +14,10 @@ Falco can send alerts to one or more channels:
...
 * A client via the gRPC API
 
-The channels are configured via the falco configuration file `falco.yaml`. See the [Falco Configuration](../configuration) page for more details. 
+The channels are configured via the falco configuration file `falco.yaml`. See the [Falco Configuration](/docs/reference/daemon/config-options/) page for more details.
 
 Find further information about how to configure each of those channels under [Alert Channels](/docs/alerts/channels/).
+
+## Integration with third parties
+
+Falco alerts can easily be forwarded to third party systems like off-host SIEM, databases or Faas. A forwarder proxy, [Falcosidekick](/doc/alerts/forwarding) was created to facilitate the integrations.
```
Comment (leogr) [inline content/en/docs/alerts/_index.md:23]:
```suggestion
Falco alerts can easily be forwarded to third-party systems like off-host SIEM, databases, or Faas. A forwarder proxy, [Falcosidekick](/doc/alerts/forwarding), was created to facilitate the integrations.
```

File: content/en/docs/alerts/channels.md [hunk 4]
```diff
@@ -144,7 +144,7 @@ http_output:
   url: http://some.url/some/path/
 ```
 
-Currently only unencrypted HTTP endpoints, and valid, secure HTTPS endpoints are supported (i.e. invalid or self signed certificates are not supported).
+Currently only unencrypted HTTP endpoints and valid HTTPS endpoints are supported (i.e. invalid or self signed certificates are not supported).
```
Comment (leogr) [inline content/en/docs/alerts/channels.md:147]:
```suggestion
Currently, only unencrypted HTTP endpoints and valid HTTPS endpoints are supported (i.e., invalid or self-signed certificates are not supported).
```

File: content/en/docs/alerts/channels.md [hunk 5]
```diff
@@ -154,29 +154,44 @@ For all output channels, you can switch to JSON output either in the configurati
 * `rule`: the rule that resulted in the alert.
 * `priority`: the priority of the rule that generated the alert.
 * `output`: the formatted output string for the alert.
+* `hostname`: the name of the host running Falco (can be the hostname inside the container).
+* `tags`: the list of tags associated to the rule.
```
Comment (leogr) [inline content/en/docs/alerts/channels.md:158]:
```suggestion
* `tags`: the list of tags associated with the rule.
```

File: content/en/docs/alerts/forwarding.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
+---
+title: Forwarding Alerts to third parties
+description: Forward Falco Alerts to third parties
+linktitle: Forwarding Alerts to third parties
+weight: 30
+---
+
+Falco alerts can easily be forwarded to third-party systems. Their JSON format allows them to be easily consumed for storage, analysis and reaction.
```
Comment (leogr) [inline content/en/docs/alerts/forwarding.md:8]:
```suggestion
Falco alerts can easily be forwarded to third-party systems. Their JSON format allows them to be easily consumed for storage, analysis, and reaction. 
```

File: content/en/docs/alerts/forwarding.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+description: Forward Falco Alerts to third parties
+linktitle: Forwarding Alerts to third parties
+weight: 30
+---
+
+Falco alerts can easily be forwarded to third-party systems. Their JSON format allows them to be easily consumed for storage, analysis and reaction. 
+
+## Falcosidekick
+
+Falcosidekick is a proxy forwarder, it acts as central point for any fleet of Falco instances using their http outputs to send their alerts.
```
Comment (leogr) [inline content/en/docs/alerts/forwarding.md:12]:
```suggestion
Falcosidekick is a proxy forwarder that serves as a hub for a fleet of Falco instances, allowing them to send alerts to many other destinations.
```

File: content/en/docs/alerts/forwarding.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+The current available outputs are chat, alert, log, storage, streaming systems, etc. The exhaustive list can be found [here](https://github.com/falcosecurity/falcosidekick/tree/master#outputs).
+
+![Falcosidekick](/docs/images/falcosidekick_forwarding.png)
+
+Falcosidekick can also add custom fields to the alerts, filter by priority and it exposes a prometheus endpoint for metrics.
```
Comment (leogr) [inline content/en/docs/alerts/forwarding.md:18]:
```suggestion
Falcosidekick can also add custom fields to the alerts, filter them by priority and expose a Prometheus metrics endpoint.
```

File: content/en/docs/alerts/forwarding.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+The full documentation and the repository of the project are [here](https://github.com/falcosecurity/falcosidekick).
```
Comment (leogr) [inline content/en/docs/alerts/forwarding.md:20]:
```suggestion
The full documentation and the project repository are [here](https://github.com/falcosecurity/falcosidekick).
```

File: content/en/docs/alerts/forwarding.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+Use the env vars to configure Falcosidekick.
+
+```shell
+docker run -d -p 2801:2801 -e SLACK_WEBHOOKURL=XXXX falcosecurity/falcosidekick:2.27.0
+```
+
+### Installation on the host
+
+Adapt the version and the architecture to your environment, you can find all the releases [here](https://github.com/falcosecurity/falcosidekick/releases).
```
Comment (leogr) [inline content/en/docs/alerts/forwarding.md:47]:
```suggestion
Adapt the version and the architecture to your environment. You can find all the releases [here](https://github.com/falcosecurity/falcosidekick/releases).
```

File: content/en/docs/alerts/forwarding.md [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+
+## Falcosidekick UI
+
+Falcosidekick comes with its own interface to visualize the events and get statistics.
+
+![Falcosidekick UI](/docs/images/falcosidekick_forwarding_ui_1.png)
+
+### Installation in Kubernetes with Helm
+
+The installation is made at same moment than Falcosidekick by adding the argument `--set falcosidekick.webui.enabled=true`
```
Comment (leogr) [inline content/en/docs/alerts/forwarding.md:84]:
```suggestion
You can install the UI at the same moment as Falcosidekick by adding the argument `--set falcosidekick.webui.enabled=true`
```

File: content/en/docs/alerts/forwarding.md [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+
+```shell
+helm install falco falcosecurity/falco \
+-n falco --create-namespace \
+--set falcosidekick.enabled=true \
+--set falcosidekick.webui.enabled=true \
+--set tty=true 
+```
+
+Then creates a port-forward to access to it: `kubectl port-forward svc falco-falcosidekick-ui 2802:2802 -n falco`. The default credentials are `admin/admin`.
```
Comment (leogr) [inline content/en/docs/alerts/forwarding.md:94]:
```suggestion
Then create a port-forward to access it: `kubectl port-forward svc falco-falcosidekick-ui 2802:2802 -n falco`. The default credentials are `admin/admin`.
```

**Review by leogr — COMMENTED** [review]
I like it!

Left a few suggestions. Mostly minor issues and nitpicking :angel: 
Otherwise, SGTM.

**Review by leogr — APPROVED** [review]

**Pull request merged**